Ports/tar/patches/ReadMe.md

```diff
@@ -0,0 +1,6 @@
+# Patches for tar on SerenityOS
+
+## `0001-Fix-savannah-bug-64441.patch`
+
+Release 1.35 has a bug in which it doesn't include libintl and libiconv properly.
+Add libiconv libraries to the build system.
```

Comment:
This doesn't match the patch that the README is generated from. We should either modify the patch directly to say that or just leave the autogenerated README as-is.